examples/kitchen-sink/src/pages/basic-page-query.mdx

```diff
@@ -1,5 +1,13 @@
+---
+my: frontmatter
+---
+
 import { graphql } from 'gatsby';
 
+export const frontmatter = {
+  other: 'frontmatter'
```

Comment:
if there's going to be frontmatter here, I'd love for it to be used in the query and layout or something as well to solidify the use-case.